src/utils/local.ts

```diff
@@ -127,8 +127,9 @@ export class LocalRelease extends EventEmitter implements Release {
       glob(file, {}, (error, filesFound) => {
         if (error) throw error
 
-        filesFound = filesFound.map((e) => this.removePrefix(e))
-        filesFound.forEach((file) => {
+        filesFound
+					.map((e) => this.removePrefix(e))
```

Comment:
What happened to the indentation here!? haha!

Reply:
Oh god! Vim got confused 😆. I will see what files need to be changed and run prettier on them.